requirements/test.txt

```diff
@@ -9,12 +9,12 @@ anyjson==0.3.3            # via -r requirements/base.txt, kombu
 astroid==2.4.2            # via pylint
 billiard==3.3.0.23        # via -r requirements/base.txt, celery
 celery==3.1.26.post2      # via -c requirements/constraints.txt, -r requirements/base.txt
-certifi==2020.6.20        # via -r requirements/base.txt, requests
-chardet==3.0.4            # via -r requirements/base.txt, requests
-coverage==5.3             # via -r requirements/test.in
-django==2.2.16            # via -c requirements/constraints.txt, -r requirements/base.txt, jsonfield2
+certifi==2020.12.5        # via -r requirements/base.txt, requests
```

Comment:
Are the tests run under these requirements? @ivanvgh

Reply:
Coudn't run them, there is the same error that we saw.